packages/lib/src/components/Card/components/CardInput/components/ExpirationDate.tsx

```diff
@@ -57,15 +58,13 @@ export default function ExpirationDate(props: ExpirationDateProps) {
                     }
                 )}
             />
-            <div className={classNames(
-                'adyen-checkout__field__exp-date_hint_wrapper',
-                [styles['checkout__field__exp-date_hint_wrapper']],
-                {
-                    'adyen-checkout__field__exp-date_hint_wrapper--hidden': error || isValid,
-                }
-            )}>
+            <div
+                className={classNames('adyen-checkout__field__exp-date_hint_wrapper', [styles['checkout__field__exp-date_hint_wrapper']], {
+                    'adyen-checkout__field__exp-date_hint_wrapper--hidden': error || isValid
+                })}
+            >
                 <img
-                    src="https://checkoutshopper-test.adyen.com/checkoutshopper/images/components/expiry_date_hint.svg"
+                    src={`${loadingContext}images/components/expiry_date_hint.svg`}
```

Comment:
We should handle this with .getImage() and never directly with URL
Going this way we make it harder for the CDN changes.

Reply:
Fair enough!